virtual_ecosystem/models/animals/animal_communities.py

```diff
@@ -133,6 +142,9 @@ def migrate_community(self) -> None:
     def die_cohort(self, cohort: AnimalCohort) -> None:
         """The function to change the cohort status from alive to dead.
 
+        This method is a computation question, not a scientific one. The science is in
+        the various methods that call die_cohort after a mortality inflicting event.
+
```

Comment:
This is a bit obscure. Also generally avoid "The function to" or "A method that" etc in docstring titles.

```suggestion
        """Remove a dead cohort  from a community.
        
        This method is used to mark a cohort as dead and remove it a community by 
        any of the methods that can inflict mortality.
```

I guess we could rename it `remove_dead_cohort` for further clarity?